Tests/CodegenCLITests/Commands/InitializeTests.swift

```diff
@@ -113,14 +113,32 @@ class InitializeTests: XCTestCase {
   func test__validation__givenModuleType_other_withNoTargetName_shouldNotThrow() throws {
     // given
     let options = [
-      "--schema-name=MySchemaName",
+      "--schema-namespace=MySchemaName",
       "--module-type=other",
     ]
 
     // then
     expect { try self.parse(options) }.notTo(throwError())
   }
 
+  func test__validation__givenBothSchemaNamespaceOptions_shouldThrow() throws {
```

Comment:
I would have liked to have a test to ensure that `--schema-name` can be used but it seems that swift-argument-parser actually injects options into the command line for properties with initialized values, like `schemaNamespace` now is. So when I wrote that test it fails with the validation that you cannot use both options at the same time. 😢